pygmt/src/meca.py

```diff
@@ -273,9 +273,9 @@ def meca(
         if depth is not None:
             spec["depth"] = np.atleast_1d(depth)
         if plot_longitude is not None:  # must be in string type
```

Comment:
Remove the comment here?
```suggestion
        if plot_longitude is not None:
```